source/characterProcessing.py

```diff
@@ -34,6 +34,8 @@
 class LocaleDataMap(Generic[_LocaleDataT], object):
 	"""Allows access to locale-specific data objects, dynamically loading them if needed on request"""
 
+	_noDataLocalesCache: set[str]
```

Comment:
is there a reason why this was added to the class rather than kept in `__init__`?

Reply:
This is only a type hint, the set itself is initiated in `__init__`

Reply:
normally the type hint would go in `__init__` though right?